transport/socks5/stream_dialer_test.go

```diff
@@ -167,17 +167,19 @@ func testExchange(tb testing.TB, listener *net.TCPListener, destAddr string, req
 }
 
 func TestConnectWithoutAuth(t *testing.T) {
+	var running sync.WaitGroup
```

Comment:
Move this to just before the `go func()`, so it's clearer you are adding that goroutine